### ISSUE TITLE
Android Emulator does not start on the Linux VM

### ISSUE THREAD

**Issue by JoyMitra** [idanr1986/cuckoo-droid#47]
Hi,

When I try to add a system image to my avd I get the following error:

"your cpu doesn't support required features (vt-x or svm) android studio in ubuntu virtual machine"

This does not happen when I am creating an AVD with the same system image on my host machine. It only happens on the VM. Is there a way to enable virtualization for the CPU when running the Linux VM. The documentation for Cuckoo droid says that no special configuration is required for setting up the Guest VM. So I am not sure why I am facing this problem.

Any help will be appreciated.

**Comment by JoyMitra** [comment]
It seems like the problem is with VirtualBox. VirtualBox disables hardware virtualization. This is why the emulators are unable to start. However, the whole documentation of cuckoodroid is written with VirtualBox. Can someone confirm if muy analysis is correct? Or am I missing to configure something in VirtualBox. Will it help to make paravirtualization=minimal?

**Comment by wickywaka** [comment]
I gave up on this. Now I am trying to use AVD on host machine but still no success.
Did you managed to make cuckoo-droid work with avd on host?
I tried using cuckoo 1.2, but it does't go further than the message `INFO: Task #2: acquired machine aosx_1`.

**Comment by wickywaka** [comment]
I am not sure, does the emulator is supposed to open, when we run the cuckoo or it stays hidden. I dont't know if there are headless or gui modes for avd.

**Comment by JoyMitra** [comment]
I think the avd is supposed to show up. The documentation says it needs to stay on during analysis. The headless and gui modes are for whether you want to see the steps in action when the analysis is performed by the VM

**Comment by JoyMitra** [comment]
@waqarrashid33 did you manage to get cuckoodroid working by setting up the emulator on the host?

**Comment by wickywaka** [comment]
Nope. Now I am trying to use the Android x86 image on virtualbox. It starts the vm but doesn't work. You can see the details of the problem I am facing here #48

**Comment by wickywaka** [comment]
@JoyMitra I managed to launch the avd when I submit an apk file but still getting the timeout error, I opened a ticket for it here #50. How is your progress?

**Comment by JoyMitra** [comment]
@waqarrashid33 I managed to launch the avd in the host machine. I start the server and submit an apk. But the analysis keeps running.

**Comment by wickywaka** [comment]
Did you checked the traffic on Wireshark? Seems kind of same problem for me.

**Comment by wickywaka** [comment]
Any updates?
